index.js

```diff
@@ -101,6 +114,21 @@ const ignored = [
     property: "position",
     value: "fixed",
     ignoredProperties: ["float", "clear", "vertical-align"]
+  },
+  {
+    property: "flex-basis",
```

Comment:
> width/height have no effect on this element since flex-basis is already defined

https://github.com/captainbrosset/useless-css-properties/blob/master/rules.md#grid--flex-properties

Reply:
not sure if `max-width` and `max-height` should also be listed here?